services/ymax-planner/test/deposit-tools.test.ts

```diff
@@ -27,6 +31,44 @@ const feeBrand = Far('fee brand (BLD)') as Brand<'nat'>;
 
 const powers = { fetch, setTimeout };
 
+/**
+ * Back-compat utility used by CLI or handlers
+ * @deprecated in favor of planDepositToAllocations
```

Comment:
no longer "used by CLI or handlers" right?


```suggestion
 * Helper for test results
```